### PR TITLE
Possibility to hide existed menu entries #4558

### PR DESCRIPTION
**Pull request by smirnovigor**
Signed-off-by: smirnovigor smirnov83@gmail.com
- [x] Implementation
- [ ] Example
- [ ] Documentation
- [ ] Integration tests (stefw can help with this)

### CODE REVIEW

**Comment by stefwalter** [comment]
What does this look like in practice? Could you provide an example? Once we're good with that, the documentation also needs to be updated.

**Comment by stefwalter** [comment]
Looks like this needs fixing:

```
pkg/shell/base_index.js:798:15: Missing semicolon.
```

File: pkg/shell/base_index.js [hunk 1]
```diff
@@ -779,9 +781,21 @@ define([
                         item.path = name + "/" + item.path;
                     if (item.path.slice(-6) == "/index")
                         item.path = item.path.slice(0, -6);
+
+                    if(info.override){
```
Comment (stefwalter) [inline pkg/shell/base_index.js]: Could you add a space before the bracket?

File: pkg/shell/base_index.js [hunk 2]
```diff
@@ -779,9 +781,21 @@ define([
...
+                        overrides.push(info.override);
+                    }
+
                     self.items[item.path] = item;
                 });
             });
+
+            // override menu entries, mark them
+            $.each(overrides, function(index, path){
+                if(self.items[path]){
```
Comment (stefwalter) [inline pkg/shell/base_index.js]: Could you add spaces before the brackets on the two above lines?

Comment (smirnovigor) [inline pkg/shell/base_index.js]: sure

Comment (smirnovigor) [inline pkg/shell/base_index.js]: done

**Comment by smirnovigor** [comment]
An example of [new manifest.json file](https://github.com/cockpit-project/cockpit/issues/4558#issuecomment-226415591)

**Comment by stefwalter** [comment]
Some discussion in #4558 that we should resolve before merging this or an alternative.

**Pull request closed without merging**